.env.example

```diff
@@ -0,0 +1,20 @@
+#Postgres environment variable
+
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=
+POSTGRES_DB=um_db
+UM_PASSWORD=
+
+
+#User-Manager environment variable
```

Comment:
These comments should be located in readme

Reply:
Add comment to README.md